lib/matplotlib/axes/_axes.py

```diff
@@ -4995,13 +5002,37 @@ def fill_betweenx(self, y, x1, x2=0, where=None,
                 continue
 
             N = len(yslice)
-            Y = np.zeros((2 * N + 2, 2), float)
+            Y = np.zeros((2 * N + 2, 2), np.float)
+            if interpolate:
+                def get_interp_point(ind):
```

Comment:
Is this the exact same function as in `fill_between`?
If so, I think it should be factored out and moved to the cbook module as a private function.